api/src/models/JobMetadata.ts

```diff
@@ -1,12 +1,16 @@
 import PageOrder from './PageOrder';
 import DocumentOrder from './DocumentOrder';
+import AudioOrder from './AudioOrder';
 import Job from './Job';
 
 class JobMetadata {
     job: Job;
+    _filename: string;
     _order: PageOrder = null;
-    _documents: DocumentOrder = null
-    published: boolean = false;
+    _documents: DocumentOrder = null;
+    _audio: AudioOrder = null;
+    _published: Boolean = false;
+    published: Boolean = false;
```

Comment:
Why do you have _published AND published here?

I think the general rule is to use the underscore for private/internal properties, and no underscore for public ones. I think in this case, you can just make it a public property, as there is no need to associate logic with it. (The method in the Ruby version of the code is just there to initialize the variable to false, but having a default value in the declaration here does the job just as well, if not better).